multinode/roles/monitoring_install/tasks/main.yml

```
@@ -0,0 +1,149 @@
+  systemd:
+    name: node-exporter
+    state: started
+    enabled: yes
+
+- name: install unzip
+  apt: 
+    name: unzip
+    state: present
+  become: true
+
+- name: Install promtail 
```

yamllint

[error] trailing spaces

(trailing-spaces)